src/defs.h

```diff
@@ -674,6 +675,9 @@ struct user_params_t {
     uint32_t dummy_mps;                   // client side only
     TicksDuration dummySendCycleDuration; // client side only
     uint32_t rate_limit;
+    bool sockperfDumpDataToFile;
+    std::string sockperfDumpDataDirectory;
+	uint32_t syncFlag;
```

Comment:
Rename syncFlag to fileOpenMode. Flag reflects a boolean variable

Reply:
Done